transcriptic/commands.py

```diff
@@ -1307,10 +1366,10 @@ def get_project_id(api, name):
 
 
 def get_project_name(api, id):
```

Comment:
I am assuming we will add tests for this method, later. Wondering if we really need this method.

Reply:
hmm... yea... seems like this is only called in one place. We could probably extract it